inscriptions/ISic030002.xml

```diff
@@ -0,0 +1,412 @@
+                                <geo>37.058593475, 14.896706425</geo>
+                            </origPlace>
+                            <origDate datingMethod="#julian" notBefore-custom="-0525" notAfter-custom="-0500">525-500 BC</origDate> <!-- previous editors dating: post 491 - ante 485; Manganaro: 450ca.-->
+                        </origin>
+                        <provenance type="found">Found in the area of Acrae<geo>37.058593475, 14.896706425</geo>.</provenance>
+                        <provenance type="observed" subtype="autopsied">No Autopsy</provenance>
+                        <acquisition>New York, The Metropolitan Museum (1925)</acquisition>
+                    </history>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>Encoded following EpiDoc guidelines 9.2</p>
+            <xi:include href="../alists/ISicily-taxonomies.xml">
+                <xi:fallback>
+                    <p>Taxonomies for ISicily controlled values</p>
+                </xi:fallback>
+            </xi:include>
+        </encodingDesc>
+        <profileDesc>
+            <calendarDesc>
+                <calendar xml:id="julian">
+                    <p>Julian Calendar</p>
+                </calendar>
+            </calendarDesc>
+            <langUsage>
+                <language ident="en">English</language> 
+                <language ident="it">Italian</language> 
+                <language ident="grc">Ancient Greek</language> 
+                <language ident="la">Latin</language> 
+                <language ident="he">Hebrew</language> 
+                <language ident="phn">Phoenician</language>
+                <language ident="xpu">Punic</language>
+                <language ident="osc">Oscan</language> 
+                <language ident="xly">Elymian</language> 
+                <language ident="scx">Sikel</language>  
+                <language ident="sxc">Sikan</language>  
+            </langUsage>
+            <textClass>
+                <keywords scheme="http://www.eagle-network.eu/voc/typeins.html">
+                    <term ana="#function.decree" ref="https://www.eagle-network.eu/voc/typeins/lod/384.html">decree</term>
+                </keywords>
+            </textClass>
+        </profileDesc>
+        <revisionDesc status="edited">
+            <listChange>
+                <change when="2020-10-05" who="#SS">Simona Stoyanova normalised Unicode</change>
+                <change when="2020-10-08" who="#SS">Simona Stoyanova updated list of languages</change>
+                <change when="2021-04-08" who="#VM">Valentina Mignosa added metadata, text</change>
+            </listChange>
+        </revisionDesc>
+    </teiHeader>
+    <facsimile>
+        <surface/> 
+        <surface type="front">
+            <graphic n="screen" url="https://www.metmuseum.org/art/collection/search/251480" >
```

Comment:
leave this for now, but it will need to be downloaded and converted to a tiled tif, etc. - so send me a separate note about this, with details of the images to be used.